Mathlib/Data/PFunctor/Multivariate/M.lean

```diff
@@ -125,21 +119,18 @@ without its contents -/
 def M.corecShape {β : Type u} (g₀ : β → P.A) (g₂ : ∀ b : β, P.last.B (g₀ b) → β) :
     β → P.last.M :=
   PFunctor.M.corec fun b => ⟨g₀ b, g₂ b⟩
-
 set_option linter.uppercaseLean3 false in
 #align mvpfunctor.M.corec_shape MvPFunctor.M.corecShape
 
 /-- Proof of type equality as an arrow -/
-def castDropB {a a' : P.A} (h : a = a') : P.drop.B a ⟹ P.drop.B a' := fun _i b => Eq.recOn h b
-
+def castdropB {a a' : P.A} (h : a = a') : P.drop.B a ⟹ P.drop.B a' := fun _i b => Eq.recOn h b
```

Comment:
Why was this lowercased? It seems to already be matching conventions.

Reply:
Sorry, I mistook.